drivers/disk/usdhc.c

```diff
@@ -2534,20 +2534,22 @@ static int usdhc_sd_init(struct usdhc_priv *priv)
 		usdhc_set_bus_width(base, USDHC_DATA_BUS_WIDTH_4BIT);
 	}
 
-	/* set sd card driver strength */
-	ret = usdhc_select_fun(priv, SD_GRP_DRIVER_STRENGTH_MODE,
-		priv->card_info.driver_strength);
-	if (ret) {
-		LOG_ERR("Set SD driver strehgth failed: %d\r\n", ret);
-		return ret;
-	}
+	if (priv->card_info.version >= SD_SPEC_VER3_0) {
+		/* set sd card driver strength */
+		ret = usdhc_select_fun(priv, SD_GRP_DRIVER_STRENGTH_MODE,
+			priv->card_info.driver_strength);
+		if (ret) {
+			LOG_ERR("Set SD driver strehgth failed: %d\r\n", ret);
```

Comment:
should be `LOG_ERR("Set SD driver strength failed: %d", ret);`, same below

Reply:
It seems that all LOG messages have \r\n, I have removed them all

Reply:
There's also a typo in the message, "strength" is misspelled.